### ISSUE TITLE
a11y violation

### ISSUE THREAD

**Issue by gcattan** [react-component/tree#546]
Hi, 

There is a minor a11y violation in your code:

![image](https://user-images.githubusercontent.com/6229031/158361156-79ad035c-1728-44ab-90f8-93fb9d353e9b.png)

Is it possible to do a fix?

**Comment by carrn7k** [comment]
I agree. Right now the tree is not very accessible. Adding a label, and possibly a prop to pass to that label, would make the tree much more accessible.
